test/format/smoke.sh

```diff
@@ -19,3 +19,5 @@ args="$args runs.threads=4 "
 $TEST_WRAPPER ./t $args runs.type=row
 # Force a rebalance to occur with statistics logging to test the utility
 $TEST_WRAPPER ./t $args runs.type=row statistics.server=1 ops.rebalance=1
+# Force a block incremental backup
+$TEST_WRAPPER ./t $args backup=1 backup.incremental=block
```

Comment:
@sueloverso, the `smoke.sh` script only runs for 50K ops with 4 threads, so this might not run long enough to perform a backup?

Reply:
I removed the change to smoke.sh as I have not been able to consistently trigger the original failure.